tcp_evaluation.py

```diff
@@ -90,13 +91,19 @@ def cleanup_containers(docker_traffic_gen, docker_traffic_receiver):
     )
 
     bg_traffic_gen, bg_traffic_receiver, file_sender, file_receiver = connect_containers()
+    mininet_connection = rpyc.connect(ni.ifaddresses('docker0')[ni.AF_INET][0]['addr'], 18861)
 
+    results = []
     for i in range(args.number_of_runs):
         # Start the file receiver in stream mode
         logs_receiver = file_receiver.exec_run(f"python3 /home/app/server_socket.py", stream=True)
 
+        for line in logs_receiver.output:
+            if line == b"INFO:root:Waiting for a connection...\n":
```

Comment:
I figured out that probably it remains blocked in this control here, if I try to print that line before the control it seems that it is empty. Probably it is the reason why the script remains stucked after the container connection